### ISSUE TITLE
devspace render: "\n" missing between documents in manifest

### ISSUE THREAD

**Issue by shaunc** [devspace-cloud/devspace#943]
**What happened?**  

With 2 deployments, the resulting combined manifest produced contains no "\n" before document separator.

For example. In otherwise empty directory:
```
% helm create foo
```
Then in ./devspace.yaml, add:
```
version: v1beta6
deployments:
- name: foo
  helm:
    chart:
      name: ./foo
- name: bar
  helm:
    chart:
      name: ./foo
```
The result of `devspace render` is below. Note there is no newline at end. But in particular, look at line 82 -- spippet:
```
          resources:
            {}---
# Source: foo/templates/serviceaccount.yaml
```
Full render output:
```
---
# Source: foo/templates/serviceaccount.yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: foo
  labels:

    helm.sh/chart: foo-0.1.0
    app.kubernetes.io/name: foo
    app.kubernetes.io/instance: foo
    app.kubernetes.io/version: "1.16.0"
    app.kubernetes.io/managed-by: Helm
---
# Source: foo/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: foo
  labels:
    helm.sh/chart: foo-0.1.0
    app.kubernetes.io/name: foo
    app.kubernetes.io/instance: foo
    app.kubernetes.io/version: "1.16.0"
    app.kubernetes.io/managed-by: Helm
spec:
  type: ClusterIP
  ports:
    - port: 80
      targetPort: http
      protocol: TCP
      name: http
  selector:
    app.kubernetes.io/name: foo
    app.kubernetes.io/instance: foo
---
# Source: foo/templates/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: foo
  labels:
    helm.sh/chart: foo-0.1.0
    app.kubernetes.io/name: foo
    app.kubernetes.io/instance: foo
    app.kubernetes.io/version: "1.16.0"
    app.kubernetes.io/managed-by: Helm
spec:
  replicas: 1
  selector:
    matchLabels:
      app.kubernetes.io/name: foo
      app.kubernetes.io/instance: foo
  template:
    metadata:
      labels:
        app.kubernetes.io/name: foo
        app.kubernetes.io/instance: foo
    spec:
      serviceAccountName: foo
      securityContext:
        {}
      containers:
        - name: foo
          securityContext:
            {}
          image: "nginx:1.16.0"
          imagePullPolicy: IfNotPresent
          ports:
            - name: http
              containerPort: 80
              protocol: TCP
          livenessProbe:
            httpGet:
              path: /
              port: http
          readinessProbe:
            httpGet:
              path: /
              port: http
          resources:
            {}---
# Source: foo/templates/serviceaccount.yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: bar-foo
  labels:

    helm.sh/chart: foo-0.1.0
    app.kubernetes.io/name: foo
    app.kubernetes.io/instance: bar
    app.kubernetes.io/version: "1.16.0"
    app.kubernetes.io/managed-by: Helm
---
# Source: foo/templates/service.yaml
apiVersion: v1
kind: Service
metadata:
  name: bar-foo
  labels:
    helm.sh/chart: foo-0.1.0
    app.kubernetes.io/name: foo
    app.kubernetes.io/instance: bar
    app.kubernetes.io/version: "1.16.0"
    app.kubernetes.io/managed-by: Helm
spec:
  type: ClusterIP
  ports:
    - port: 80
      targetPort: http
      protocol: TCP
      name: http
  selector:
    app.kubernetes.io/name: foo
    app.kubernetes.io/instance: bar
---
# Source: foo/templates/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: bar-foo
  labels:
    helm.sh/chart: foo-0.1.0
    app.kubernetes.io/name: foo
    app.kubernetes.io/instance: bar
    app.kubernetes.io/version: "1.16.0"
    app.kubernetes.io/managed-by: Helm
spec:
  replicas: 1
  selector:
    matchLabels:
      app.kubernetes.io/name: foo
      app.kubernetes.io/instance: bar
  template:
    metadata:
      labels:
        app.kubernetes.io/name: foo
        app.kubernetes.io/instance: bar
    spec:
      serviceAccountName: bar-foo
      securityContext:
        {}
      containers:
        - name: foo
          securityContext:
            {}
          image: "nginx:1.16.0"
          imagePullPolicy: IfNotPresent
          ports:
            - name: http
              containerPort: 80
              protocol: TCP
          livenessProbe:
            httpGet:
              path: /
              port: http
          readinessProbe:
            httpGet:
              path: /
              port: http
          resources:
            {}
```

**What did you expect to happen instead?**  

The documents in the manifest should be properly separated.

**How can we reproduce the bug?** (as minimally and precisely as possible)  

Above should suffice.

**Local Environment:**  
- Operating System:  mac
- Deployment method: helm

**Kubernetes Cluster:**  
- Cloud Provider: NA
- Kubernetes Version: NA

**Anything else we need to know?**  

/kind bug

**Comment by FabianKramm** [comment]
@shaunc thanks for opening this issue! Yes you are correct, this should be easily fixed and we'll add this to the release today!

**Comment by FabianKramm** [comment]
Should be fixed in v4.5.1
